consensus/istanbul/validator/doc.go

```diff
@@ -14,5 +14,40 @@
 // You should have received a copy of the GNU Lesser General Public License
 // along with the klaytn library. If not, see <http://www.gnu.org/licenses/>.
 
-// Package validator implements the types related to the validators participating in istanbul consensus.
+/*
+Package validator implements the types related to the validators participating in consensus.
+
+Types in validator package implement `Validator` and `ValidatorSet` interface in istanbul/validator.go file.
+Those types are used for validating blocks to make consensus.
```

Comment:
How about changing above two lines like below?

```
`Validator` and `ValidatorSet` interfaces in validator package are used to validate blocks and make a consensus.
Implementations of these interfaces are different, depending on proposer-choosing policy.
```

Reply:
thanks! it looks much better. I've just removed `in validator package` because the interfaces are not in validator package. they are in istanbul package.